contracts/utils/contracts/src/Refundable.sol

```diff
@@ -22,32 +22,58 @@ pragma solidity ^0.5.9;
 contract Refundable {
 
     // This bool is used by the refund modifier to allow for lazily evaluated refunds.
-    bool internal shouldNotRefund;
+    bool internal _shouldNotRefund;
 
     modifier refundFinalBalance {
         _;
-        if (!shouldNotRefund) {
-            _refundNonzeroBalance();
-        }
+        _refundNonZeroBalanceIfEnabled();
     }
 
     modifier disableRefundUntilEnd {
-        if (shouldNotRefund) {
+        if (_areRefundsDisabled()) {
             _;
         } else {
-            shouldNotRefund = true;
+            _disableRefund();
             _;
-            shouldNotRefund = false;
-            _refundNonzeroBalance();
+            _enableRefund();
```

Comment:
This is probably as readable as it gets, but considering inlining, I think that it probably makes sense to put the functionality of calling

```
_enableRefund();
_refundNonZeroBalance();
```

into a single internal function to save a little bit more on codesize.

Reply:
Nice, saved about 10 bytes :p

Reply:
Every little bit counts I guess haha